code/modules/mob/new_player/new_player.dm

```diff
@@ -298,7 +298,7 @@
 
 /mob/new_player/proc/DiseaseCarrierCheck(var/mob/living/carbon/human/H)
 	// 10% of players are joining the station with some minor disease
-	if(prob(10))
+	if(prob(10) || isvox(H) && prob(10))
```

Comment:
This doesn't do what you think it does. What this is going to do is make Vox have the same chance to have a minor disease as before (10%), but lower the chance for humans to arrive carrying a disease to 1%.

Either bad code or trying to sneak a nerf to new viro under a different name.

Reply:
This is incorrect. It doesn't change the normal chance but it rolls twice for vox, which if I didn't fuck up due to being tired would give them a 19% chance

Reply:
Wrong button
Anyway I made a comment here before but it was wrong so ignore it